frame/support/src/traits.rs

```diff
@@ -1664,6 +1665,70 @@ pub trait Instance: 'static {
     const PREFIX: &'static str ;
 }
 
+/// A trait similar to `Convert` to convert values from `B` (i.e. the `Balance` type of the chain)
+/// into u64, which is the standard numeric types used in election and other places where complex
+/// calculation over balance type is needed.
```

Comment:
Should the trait docs hint what to do with overflows? It seems that this might be a lossy conversion and it's not clear to me how to implement such trait for say `U256` balance type just by looking at the trait definition.

Reply:
I think end of the day that depends on the chain's logic and their builders decision.

In other words, I cannot give them a definitive advise here of what to do :D so I will just leave it open.